boggs/src/main/java/com/hartwig/healthchecks/boggs/healthcheck/prestast/PrestatsExtractor.java

```diff
@@ -33,43 +32,29 @@
 
 	public PrestatsReport extractFromRunDirectory(String runDirectory) throws IOException, EmptyFileException {
 		List<PrestatsDataReport> summaryData = getSummaryFilesData(runDirectory);
-		List<PrestatsDataReport> fastqcData = getfastqFilesData(runDirectory);
+		PrestatsDataReport fastqcData = getfastqFilesData(runDirectory);
 
-		if (summaryData.isEmpty() || fastqcData.isEmpty()) {
+		if (summaryData.isEmpty() || fastqcData == null) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ['if' have incorrect indentation level 16, expected level should be 8.](https://www.codacy.com/app/hmf/health-checks/file/2780521380/issues/source?bid=3363458&fileBranchId=3365368#l37)